JumpscaleLibs/tools/threebot_proxy/threebot_proxy.py

```diff
@@ -0,0 +1,101 @@
+import nacl.encoding
+import requests
+
+from bottle import redirect, request, abort
+from functools import wraps
+from urllib.parse import urlencode
+
+from Jumpscale import j
+
+try:
+    from beaker.middleware import SessionMiddleware
+except (ModuleNotFoundError, ImportError):
+    j.builders.runtimes.python3.pip_package_install("beaker")
+    from beaker.middleware import SessionMiddleware
+
+
+session_opts = {"session.type": "file", "session.data_dir": "./data", "session.auto": True}
```

Comment:
why not make it private, seems to be a pretty local constant
```suggestion
_session_opts = {"session.type": "file", "session.data_dir": "./data", "session.auto": True}
```